br/tests/lightning_partition_incremental/run.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright 2022 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+run_lightning --backend=local -d "tests/$TEST_NAME/data"
+
+LOG_FILE1="$TEST_DIR/lightning_partition_incremental1.log"
+LOG_FILE2="$TEST_DIR/lightning_partition_incremental2.log"
+
+run_lightning --log-file "$LOG_FILE1" --config "tests/$TEST_NAME/config1.toml" -d "tests/$TEST_NAME/data1" &
+pid1="$!"
+run_lightning --log-file "$LOG_FILE2" --config "tests/$TEST_NAME/config2.toml" -d "tests/$TEST_NAME/data2" &
+pid2="$!"
+
+# Must wait for pid explicitly. Otherwise, we can't detect the process is exited with error.
+wait "$pid1" "$pid2"
+
+grep -F 'checksum pass' "$LOG_FILE1" | grep -Fq 'auto_rowid'
+grep -F 'checksum pass' "$LOG_FILE1" | grep -Fq 'non_pk_auto_inc'
+
+run_sql 'admin check table incr.auto_rowid'
+run_sql 'admin check table incr.non_pk_auto_inc'
+
+run_sql 'select count(*) from incr.auto_rowid'
```

Comment:
Please also check that the are no tidb_rowid duplicate across all partitions.

Reply:
Fixed in https://github.com/pingcap/tidb/pull/35905/commits/729a523818e1c6db14923811196dde384ca58163.